content/docs/concepts/components/buildpack.md

```diff
@@ -5,33 +5,34 @@ weight=2
 
 ## What is a buildpack?
 
-A buildpack is a unit of work that inspects your app source code and formulates a plan to build and run your
+A buildpack is a set of scripts that inspects your app source code and creates a plan to build and run your
 application.
 
 <!--more-->
 
-Typical buildpacks are a set of at least three files:
+Typical buildpacks consist of at least three files:
 
 * `buildpack.toml` -- provides metadata about your buildpack
 * `bin/detect` -- determines whether buildpack should be applied
-* `bin/build` -- executes buildpack logic
+* `bin/build` -- executes build logic
 
 #### Meta-buildpack
 
 There is a different type of buildpack commonly referred to as a **meta-buildpack**. It contains only a 
 `buildpack.toml` file with an `order` configuration that references other buildpacks. This is useful for 
 composing more complex detection strategies.
 
-## Anatomy of a buildpack
+## Buildpack phases
 
-There are two essential phases that allow buildpacks to create a runnable image.
+These are two essential steps that allow buildpacks to create a runnable image.
```

Comment:
```suggestion
There are two essential steps that allow buildpacks to create a runnable image.
```

Reply:
I intentionally wrote this way as I thought that it would not repeat that is already stated in the header, but I am not proficient at the language. ) Fixed.